detox/test/e2e/14.network.test.js

```diff
@@ -48,4 +48,21 @@ describe('Network Synchronization', () => {
 
     await device.setURLBlacklist([]);
   });
+
+  it('launchArgs with detoxURLBlacklistRegex should set the blacklist', async () => {
+    await device.launchApp({
+      newInstance: true,
+      launchArgs: { detoxURLBlacklistRegex: ' \\("http://192.168.1.253:19001/onchange","https://e.crashlytics.com/spi/v2/events",".*localhost.*"\\)' },
```

Comment:
Switch the localhost expression with a fancier one that would match `localhost:9000`, but also include common reg-exp standards translation pitfalls?